MWCamera/Classes/MWBaseCameraViewController.swift

```diff
@@ -196,7 +196,10 @@ extension MWBaseCameraViewController {
         }
 
         guard videoDevice.activeVideoMinFrameDuration.timescale != frameRate,
```

Comment:
Although this addition fixes the bug of an uninitialized `self.frameRate` it does reveal another bug. This method is intended to fix the frame rate of videoDevice, but the guard statement will return in a situation where `videoDevice.activeVideoMixFrameDuration.timescale` may not be equal to `videoDevice.activeVideoMaxFrameDuration.timescale`.

Also if the an error is ever thrown in the do statement then `self.frameRate` won't be set as well. Here is snippet of code that solves all problems. let me know what you think.

```
        guard videoDevice.activeVideoMinFrameDuration.timescale != videoDevice.activeVideoMaxFrameDuration.timescale,
            || videoDevice.activeVideoMinFrameDuration.timescale != frameRate else {
                self.frameRate = frameRate
                return
        }

        do {
            try videoDevice.lockForConfiguration()
            videoDevice.activeVideoMinFrameDuration = CMTime.init(value: 1, timescale: CMTimeScale(frameRate))
            videoDevice.activeVideoMaxFrameDuration = CMTime.init(value: 1, timescale: CMTimeScale(frameRate))
            videoDevice.unlockForConfiguration()

            self.frameRate = frameRate
        } catch {
            print("[mwCamera]: Could not lock device for configuration: \(error)")
            self.frameRate = videoDevice.activeVideoMaxFrameDuration.timescale
        }
```

Well our problem is not really solved when the catch statement is evoked(frame rate is not fixed so frame rate logic further down is still open to video/audio sync issues). I'm not sure how I should handle that problem. Suggestions are welcomed 😄 .